unskript-ctl/unskript-client.py

```diff
@@ -1449,6 +1523,8 @@ def stop_debug():
                         help='Run the given runbook FILENAME [-RUNBOOK_PARM1 VALUE1] etc..')
     parser.add_argument('-rc', '--run-checks', type=str,
```

Comment:
run checks should support passing the parameters using the yaml file. Running suite gives the wrong impression.